serverless-architecture-on-aws/lab-1/README.md

```diff
@@ -0,0 +1,53 @@
+# Lab 1: Getting started with Pulumi Templates
+
+## Create a directory
+
+To get started, we need a directory. Run these two commands in your terminal:
+
+```bash
+mkdir my-serverless-app && cd my-serverless-app
+```
+
+## Initialise your project
+
+We're using the AWS Serverless Application template:
+
+```bash
+pulumi new serverless-aws-typescript
+```
+
+## Inspect your project
+
+Let's take a look at what we've created
+
+## Deploying your serverless app
+
+Run the following command:
+
+```bash
+pulumi up
+```
+
```

Comment:
Please select `yes` to continue

Reply:
I want them to see what's going on so I don't want them to jump the gun